analyses/telomerase-activity-prediction/07_EXTEND-at-threshold.Rmd

```diff
@@ -0,0 +1,206 @@
+---
+title: "Assess EXTEND results calculated with tumor purity thresholded data"
+author: "SJ Spielman for CCDL"
+date: "2023"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+---
+
+```{r setup, include=FALSE}
+library(tidyverse)
+theme_set(theme_bw())
+set.seed(2023)
+```
+This notebook was written as part of manuscript revisions.
+Here, we explore how EXTEND-related results might change if only tumors at a certain level of known tumor purity (`tumor_fraction` metadata variable) are considered.
+Only stranded RNA data is considered here, as in the manuscript.
+This notebook consumes result file `TelomeraseScores_PTBAStranded_FPKM_thresholded.txt` produced by this module's script `01-run-EXTEND.R` run with the ` --apply_tumor_purity_threshold` flag.
+
+The associated issue for this exploration is here: https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1625
+
+
+
+## Set directories and files
+
+```{r}
+# directories
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+analyses_dir <- file.path(root_dir, "analyses")
+extend_dir <- file.path(analyses_dir, "telomerase-activity-prediction")
+
+# input files
+metadata_file <- file.path(data_dir, "pbta-histologies.tsv")
+palette_file <- file.path(root_dir, "figures", "palettes", "broad_histology_cancer_group_palette.tsv")
+extend_file_thresholded <- file.path(extend_dir, 
+                         "results",
+                         "TelomeraseScores_PTBAStranded_FPKM_thresholded.txt")
+extend_file_original <- file.path(extend_dir, 
+                         "results",
+                         "TelomeraseScores_PTBAStranded_FPKM.txt")
```

Comment:
Possibly some editor weirdness here with the spacing? Not sure.

Reply:
Cleaned this up in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1657/commits/1b5164e14ea05429b6cdb2d1bf868f1c9f15ef2b.